src/lib/mina_metrics/prometheus_metrics/mina_metrics.ml

```diff
@@ -77,7 +77,7 @@ module TextFormat_0_0_4 = struct
       | None ->
           (label_names, label_values)
       | Some (label_name, label_value) ->
-          let label_value_str = Fmt.strf "%a" output_value label_value in
+          let label_value_str = Fmt.str "%a" output_value label_value in
```

Comment:
@querolita used to proposed this change, and it seems this would prevent CI from building due to compilation issues. Did you check your local fmt package is pinned to the one specified in `opam.export`? Thanks :)